src/components/searchbar.js

```diff
@@ -239,18 +275,11 @@ const SearchBar = React.createClass({
     const that = this
     setTimeout(() => {
       that.handleSearch()
+
+      // update the query string in the url
+      updateQueryString('d3modules', d3modules)
     })
   },
-  componentDidUpdate() {
-    if (this.refs) {
-      if (this.refs.search || this.refs.search === '') {
-        this.refs.search.value = this.props.query.text
-      }
-      if (this.refs.user && this.props.query.userRaw) {
-        this.refs.user.value = this.props.query.userRaw
-      }
-    }
-  },
```

Comment:
not removing this, just moving it up to the top of the component's class methods.